flexbox/src/main/res/values/attrs.xml

```diff
@@ -99,5 +99,8 @@ limitations under the License.
             <enum name="baseline" value="3" />
             <enum name="stretch" value="4" />
         </attr>
+
+        <attr name="layout_minWidth" format="dimension" />
```

Comment:
Updated sample app to support min-width and min-height. I chose to make this a separate commit, let me know if you think it would be batter squashed.